vc

```diff
@@ -28,6 +28,12 @@ if [ -z "$mailaddr" ]; then
 	mailaddr="$USER@$domain"
 fi
 
+if [ -x /usr/bin/rpmdev-packager ]; then
+	packager=`rpmdev-packager`
+else
+	packager="`getent passwd $UID | cut -d: -f5` <$mailaddr>"
```

Comment:
need to parse GECOS here

Reply:
Isn't that what I'm doing?

Reply:
no, you're just parsing passwd itself, not the GECOS field
 https://en.wikipedia.org/wiki/Gecos_field

Reply:
I think I got it now.